site/source/docs/porting/asyncify.rst

```diff
@@ -1,8 +1,8 @@
-.. _Asyncify:
+.. _asyncify section:
```

Comment:
Unrelated change? Also, does this work with a space in the name?

Reply:
Spaces are fine in ref names yes.


I put the reason for this in the PR description: I also had to rename the section tag for wasm_workers.rst and
asyncify.rst because the happen to collide with the ref for the
corresponding setting name.